sdk/devcenter/azure-developer-devcenter/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-02-07)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- This release updates the Azure DevCenter library to use the 2022-11-11-preview API.
```

Comment:
To note: will be adding a more detailed CHANGELOG shortly - I also need to update the .NET one.